### PR TITLE
[MIG] mrp_workorder_lot_display: Backport to 15.0

### PR DESCRIPTION
**Pull request by grindtildeath**

### CODE REVIEW

File: mrp_workorder_lot_display/i18n/it.po [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Translation of Odoo Server.
```
Comment (JuMiSanAr) [inline mrp_workorder_lot_display/i18n/it.po:1]: Do we need these translation files?

**Review by JuMiSanAr — APPROVED** [review]

**Review by gurneyalex — APPROVED** [review]

**Comment by grindtildeath** [comment]
CI must be fixed through #1173

**Pull request merged**